k8s/default-resources.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: default-limits
+spec:
+  limits:
+  - default:
+      memory: 256Mi
+      cpu: 100m
+    defaultRequest:
+      memory: 256Mi
+      cpu: 10m
```

Comment:
Why are these so low? Our app uses 350MB atm on the VM and this will easily increase because java.

Reply:
Those are the default values if you don't specify anything (which reminds me to check the app). The idea is that e.g. the default HTTP backend does not need to request 100m (which is the default if you do not specify anything), because the nodes ATM only have around 940m allocatable.